webapp/tests/test_tags.py

```diff
@@ -376,7 +377,11 @@ def test_tag_views(self):
     response = self.client.post(url + '/tagSeries', {'path': 'test.a;hello=tiger;blah=blah'})
     self.assertEqual(response.status_code, 200)
     self.assertEqual(response['Content-Type'], 'application/json')
-    self.assertEqual(response.content, json.dumps(expected, indent=2, sort_keys=True))
+    if sys.version_info[0] >= 3:
+      self.assertEqual(response.content,
+                 json.dumps(expected, indent=2, sort_keys=True).encode('utf-8'))
+    else:
+      self.assertEqual(response.content, json.dumps(expected, indent=2, sort_keys=True))
```

Comment:
Instead of having these through the whole file, can we add a helper function to use throughout the file?
```
def json_dumps(obj, *args, **kwargs):
  if sys.version_info[0] >= 3:
    return json.dumps(obj, *args, **kwargs).encode('utf-8')
  return json.dumps(obj, *args, **kwargs)
```

Reply:
See takluyver/graphite-web#1 for more on this, we should be able to handle this all in `graphite.util.json`

Reply:
👍